src/lib.rs

```diff
@@ -19,78 +23,72 @@
 //! # fn main() {}
 //! ```
 //!
-//! `#[derive(Builder)]` will automatically generate a setter method for the `ipsum` field,
-//! looking like this:
+//! ## What you get
 //!
 //! ```rust,ignore
```

Comment:
done - doc test enabled.

but I kept the allow dead_code attr, because I would like to be transparent about the code which is generated.